civis/io/_tables.py

```diff
@@ -694,21 +689,26 @@ def dataframe_to_civis(df, database, table, api_key=None, client=None,
         warnings.warn("`archive` is deprecated and will be removed in v2.0.0. "
                       "Use `hidden` instead.", FutureWarning)
 
+    delimiter = '|'
+
     with TemporaryDirectory() as tmp_dir:
         tmp_path = os.path.join(tmp_dir, 'dataframe_to_civis.csv')
-        to_csv_kwargs = {'encoding': 'utf-8', 'index': False}
+        to_csv_kwargs = {
+            'encoding': 'utf-8', 'index': False, 'header': True,
+            'sep': delimiter, 'quoting': csv.QUOTE_ALL, 'escapechar': '\\',
+            'doublequote': True
```

Comment:
doublequote defaults to True so we shouldn't need this

Reply:
I'd prefer to be explicit about this since it's to demonstrate best practices.

Reply:
By that logic should we list all of the defaults here? IMHO we should only list values we are overriding and probably have a comment that explains why we are changing the default. That makes it more clear to contributors why we are not using pandas defaults.

We can also add a `See Also` section that mentions `pandas.to_csv` so that users/contributors reference their documentation.